project/Build.scala

```diff
@@ -32,7 +32,7 @@ object Build extends Build {
 
   val basicScalacOptions = Seq(
     "-encoding", "utf8",
-    "-target:jvm-1.7",
+    "-target:jvm-1.8",
```

Comment:
This would normally be quite a change and need a major version number, but since Akka already requires java 8 and our whole raison d'etre is akka, this seems great :)